lms/djangoapps/discussion/static/discussion/js/discussion_router.js

```diff
@@ -18,7 +18,7 @@
                 initialize: function(options) {
                     Backbone.Router.prototype.initialize.call(this);
                     _.bindAll(this, 'allThreads', 'showThread');
-                    this.courseId = options.courseId;
+                    this.rootUrl = options.rootUrl;
```

Comment:
As mentioned before, this is how it should be. The only place that uses Python conventions is in the factory which is rendered by Mako.